v1/src/test/java/com/google/cloud/teleport/templates/SpannerToTextIT.java

```diff
@@ -76,15 +76,17 @@ private void testSpannerToGCSTextBase(
           paramsAdder)
       throws IOException {
     // Arrange
-    String createTableStatement =
+    List<String> statements = new ArrayList<>();
```

Comment:
why not create sql files for these as well to keep the tests consistent? Also, why is there a difference in naming convention for table names in the templates test vs spanner tests given that we are using fresh database every time?

Reply:
The CSV import export ITs are simple and had only 1 or 2 DDL statements. Creating a sql file and loading it seemed like an overkill. We can move it to resources folder if it gets big.